scripts/alternative-install-host-dependencies.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+### Install all dependencies on Ubuntu 16.04 LTS (Xenial Xerus) for RChain dev environment
+
+
+## Detect if running in docker container - setup using sudo accordingly
+if [[ $(cat /proc/self/cgroup  | grep docker) = *docker* ]]; then
+    echo "Running in docker container!"
+    sudo=""
+else
+	sudo="sudo"
+fi
+
+## Set BASH environment so it will fail properly throwing exit code
+set -euxo pipefail
```

Comment:
Move this to the top, so that if your `cat /proc/self/whatever` fails, it'll also cause the script to terminate with an error?